### DISCUSSION TITLE
RSS Feed: Posts in wrong order

### DISCUSSION THREAD

**Discussion by papanito** [thuliteio/doks#1304] [Q&A] [answered]
Currently my blog post in the rss feed are not in the correct order and I am not sure how to correct as I don't see any ` rss.xml`. This command does not work: https://github.com/thuliteio/doks/discussions/889#discussioncomment-3604137

**Comment by h-enk** [comment, marked as answer]
Doks uses Hugo’s [embedded RSS template](https://github.com/gohugoio/hugo/blob/master/tpl/tplimpl/embedded/templates/_default/rss.xml) — no customizations

To use custom templates, see [Custom templates](https://gohugo.io/templates/rss/#custom-templates)

Do you have `weight` set in the frontmatter of your posts?

**Reply by papanito** [reply]
Yep, thanks for the hint. Seems I removed the `weight` on the latest posts. Re-adding it, fixes my issues
